src/chart/generateCategoricalChart.tsx

```diff
@@ -1253,7 +1253,13 @@ export const generateCategoricalChart = ({
 
       const containerOffset = getOffset(this.container);
       const e = calculateChartCoordinate(event, containerOffset);
-      const rangeObj = this.inRange(e.chartX, e.chartY);
+
+      const element = this.container;
+      const boundingRectWidth = element?.getBoundingClientRect()?.width;
+      const { offsetWidth } = element;
+      const scale = boundingRectWidth / offsetWidth || 1;
```

Comment:
Just checking, this division is safe if it results in `NaN` correct?

Reply:
yes, line 1260 returns 1 if the div is NaN (or 0 - which I also assumed is a value that does not make sense)